mclagsyncd/mclaglink.cpp

```diff
@@ -24,17 +24,46 @@
 #include <stdlib.h>
 #include <iostream>
 #include "logger.h"
+#include "tokenize.h"
 #include "netmsg.h"
 #include "netdispatcher.h"
 #include "swss/notificationproducer.h"
 #include "mclagsyncd/mclaglink.h"
 #include "mclagsyncd/mclag.h"
 #include <set>
 #include <algorithm>
+#include "macaddress.h"
+#include <string>
+#include <iosfwd>
+#include <iostream>
+#include <sstream>
+#include "table.h"
 
 using namespace swss;
 using namespace std;
 
+void MclagLink::addVlanMbr(std::string vlan, std::string mbr_port)
+{
+    m_vlan_mbrship.emplace(vlan_mbr(vlan,mbr_port));
+}
+
+//returns 1 if present, else returns zero
+int MclagLink::findVlanMbr(std::string vlan, std::string mbr_port)
+{
+    if (m_vlan_mbrship.find(vlan_mbr(vlan,mbr_port))  != m_vlan_mbrship.end())
```

Comment:
return (m_vlan_mbrship.find(vlan_mbr(vlan,mbr_port)) != m_vlan_mbrship.end())

Reply:
thanks for comments. addressed it